bundler/lib/bundler/man/bundle.1.ronn

```diff
@@ -36,9 +36,9 @@ We divide `bundle` subcommands into primary commands and utilities:
 * [`bundle update(1)`](bundle-update.1.html):
   Update dependencies to their latest versions
 
-* [`bundle package(1)`](bundle-package.1.html):
+* [`bundle cache(1)`](bundle-cache.1.html):
   Package the .gem files required by your application into the
-  `vendor/cache` directory
+  `vendor/cache` directory (aliases: `bundle package`, `bundle pack`)
```

Comment:
To find `bundle package` in `bundle(1)` man, just added both `package` and `pack` here.